EdvClient.js

```diff
@@ -95,7 +95,7 @@ export class EdvClient {
    *   from to associate chunked data with this document.
    * @param {number} [options.chunkSize=1048576] - The size, in bytes, of the
    *   chunks to break the incoming stream data into.
-   * @param {object} [options.recipients=[]] - A set of JWE recipients
+   * @param {object[]} [options.recipients=[]] - A set of JWE recipients
```

Comment:
I'm assuming this is the right thing to do with the docs here. I thought maybe it should be `array` or `Array` ... but maybe this is better? I don't know the latest.

Reply:
Right, so, the two notations are equivalent - `{object[]}` and `{Array<object>}`. The second one exists so that you can do things like `Array<Array<number>>`